src/ds/ring_buffer_types.h

```diff
@@ -137,6 +137,12 @@ namespace ringbuffer
       "No payload specialization for this Message");
   };
 
+#define DECLARE_RINGBUFFER_MESSAGE_NO_PAYLOAD(MTYPE) \
```

Comment:
Some messages don't have a payload (e.g. `stop` or `stopped`) and clang complains about the _empty_ variables args to `DECLARE_RINGBUFFER_MESSAGE_PAYLOAD()`.

Reply:
Minor: If we can disable that warning at the macro definition point, it'd be nicer than introducing a new macro name to the API surface.

Reply:
As discussed, left it as in for now as it gives better compile-checks for payload-less messages.